pkg/storage/newStorage/storageDB.go

```diff
@@ -0,0 +1,106 @@
+package newStorage
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/google/uuid"
+	"github.com/stasBigunenko/monorepa/model"
+	"sync"
+)
+
+type StorageDB struct {
+	Data map[uuid.UUID][]byte
+	mu   sync.Mutex
```

Comment:
Here is fine but just FYI. 
You can use `RWLock` with `RLock()` on read operations which is non-blocking for parallel reads.